src/less/components/button.less

```diff
@@ -414,25 +422,13 @@
 
         &.btn-primary,
         &.btn-secondary,
-        &.btn-link {
-            &:after {
-                border-color: lighten(@brand-secondary, 40%);
-                border-left-color: @brand-secondary;
-            }
-        }
-
-        &.btn-executive-destructive {
-            &:after {
-                border-color: hsla(0, 0%, 100%, 0.5);
-                border-left-color: @brand-error;
-            }
-        }
-
+        &.btn-link,
+        &.btn-executive-destructive,
         &.btn-guiding-destructive,
         &.btn-link-destructive {
             &:after {
-                border-color: @brand-error-light;
-                border-left-color: @brand-error;
+                border-color: lighten(@brand-secondary, 40%);
+                border-left-color: @brand-secondary;
             }
         }
 
```

Comment:
The After icon size should stay the same regardless of buttons size.
![image](https://user-images.githubusercontent.com/44967773/184603898-09071a98-7b04-4e99-9ac6-0cbd49c1b328.png)